docs/pages/guides/using-firebase.md

```diff
@@ -46,215 +76,191 @@ const firebaseConfig = {
+- Want to use native SDKs in your app.
+- You have a bare React Native app with React Native Firebase already configured but are migrating to use Expo SDK.
+- Want to add [Performance Monitoring](https://rnfirebase.io/perf/usage) into your app.
 
-### Listening for Authentication
+#### Caveats
 
-We are now ready to connect the Facebook login code in our app with our Firebase Realtime Database implementation.
+React Native Firebase requires [custom native code and cannot be used with Expo Go](/workflow/expo-go/#custom-native-code).
 
-```javascript
-import { initializeApp } from 'firebase/app';
-import {
-  getAuth,
-  onAuthStateChanged,
-  FacebookAuthProvider,
-  signInWithCredential,
-} from 'firebase/auth';
-import * as Facebook from 'expo-facebook';
-
-let myApp = initializeApp(config);
-
-const auth = getAuth(myApp);
-
-// Listen for authentication state to change.
-onAuthStateChanged(auth, user => {
-  if (user != null) {
-    console.log('We are authenticated now!');
-  }
+### Install and initialize React Native Firebase
 
-  // Do other things
-});
+#### Step 1: Install expo-dev-client
 
-async function loginWithFacebook() {
-  await Facebook.initializeAsync('<FACEBOOK_APP_ID>');
+Since React Native Firebase requires custom native code and it cannot be used with Expo Go, you need to install `expo-dev-client` in your project. The development client allows you to create a custom Expo Go app that is completely configured for your project. It also allows configuring any native code required by React Native Firebase using [Config plugins](/guides/config-plugins/) without writing native code yourself.
 
-  const { type, token } = await Facebook.logInWithReadPermissionsAsync({
-    permissions: ['public_profile'],
-  });
+To install `expo-dev-client`, run the following command in your project:
 
-  if (type === 'success') {
-    // Build Firebase credential with the Facebook access token.
-    const facebookAuthProvider = new FacebookAuthProvider();
-    const credential = facebookAuthProvider.credential(token);
+<Terminal cmd={["$ npx expo install expo-dev-client"]} />
 
-    // Sign in with credential from the Facebook user.
-    signInWithCredential(auth, credential).catch(error => {
-      // Handle Errors here.
-    });
-  }
+For more information, see [Getting started with `expo-dev-client`](/development/getting-started/#installing--expo-dev-client--in-your-project).
+
+#### Step 2: Install React Native Firebase
+
+To use React Native Firebase, it is necessary to install the `@react-native-firebase/app` module. This module provides the core functionality for all other modules. You can install it using the following command:
+
+<Terminal cmd={["$ npx expo install @react-native-firebase/app"]} />
+
+#### Step 3: Add a config plugin
```

Comment:
At this point forward, can we point them to RN Firebase's docs? They include all of these steps under their Expo install section: https://rnfirebase.io/#expo

Reply:
I don't think the rnfirebase.io/#expo is well organized and provides a clear set of instructions to follow the Expo route. From what I have observed in the community, not everyone is able to follow their way of instructions.

Also, they mention using `use_frameworks` for iOS before the Expo section starts and do not refer it back in the Expo section. That is easily missed out by some devs.

Reply:
Since my previous comment, I think they recently changed the Expo section in their docs which now looks fine.

I'll update this in the new PR I'm creating. Thanks!